Backend/core/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "chase-car-dashboard-backend-core"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+readme = "README.md"
```

Comment:
Can you remove this readme line and have it still work? I don't think we need this